doc/classes/AudioStreamPlayer3D.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Plays a sound effect with directed sound effects, dampens with distance if needed, generates effect of hearable position in space.
+		By default audio is heard from the camera position. This can be changed with a [Listener3D] node.
```

Comment:
```suggestion
		By default, audio is heard from the camera position. This can be changed by adding a [Listener3D] node to the scene and enabling it by calling [method Listener3D.make_current] on it.
```